### PR TITLE
export structure & functions for relay

### PR DESCRIPTION
**Pull request by cnderrauber**

### CODE REVIEW

File: pkg/sfu/buffer/buffer.go [hunk 1]
```diff
@@ -31,19 +31,19 @@ type pendingPackets struct {
...
-	Arrival  int64
-	Packet   rtp.Packet
-	Payload  interface{}
-	KeyFrame bool
+	Head       bool
+	Arrival    int64
+	Packet     rtp.Packet
+	Payload    interface{}
+	KeyFrame   bool
+	RawPayload []byte
```
Comment (cnderrauber) [inline pkg/sfu/buffer/buffer.go:39]: Relaydown will send RawPayload directly

File: pkg/sfu/downtrack.go [hunk 2]
```diff
@@ -239,7 +236,7 @@ func (d *DownTrack) Bind(t webrtc.TrackLocalContext) (webrtc.RTPCodecParameters,
 // because a track has been stopped.
 func (d *DownTrack) Unbind(_ webrtc.TrackLocalContext) error {
 	d.bound.set(false)
-	d.receiver.DeleteDownTrack(d.peerID)
+	d.receiver.DeleteDownTrack(d.ID())
```
Comment (cnderrauber) [inline pkg/sfu/downtrack.go:239]: TrackID in lk-server is global unique, there is no need of peerID

Comment (davidzhao) [reply pkg/sfu/downtrack.go:239]: It's globally unique per *track*, but not unique across downtracks.. For example, an UpTrack may come in as `TR_123`, all of the downtracks will inherit that track id. We'll still need to use the peerID to identify which peer the downtrack is for.

File: pkg/sfu/buffer/buffer.go (hunk 1 above)
Comment (boks1971) [reply pkg/sfu/buffer/buffer.go:39]: Nice!!!

Maybe call it `RawPacket` as there is `Payload` inside `rtp.Packet` which does not include the RTP header bytes. Just to make it clear, suggesting this change.

File: pkg/sfu/downtrack.go [hunk 3]
```diff
@@ -28,7 +28,6 @@ type TrackSender interface {
 	// ID is the globally unique identifier for this Track.
 	ID() string
 	SetTrackType(isSimulcast bool)
-	PeerID() string
```
Comment (boks1971) [inline pkg/sfu/downtrack.go:31]: This method is needed by `StreamAllocator` also. Please leave it in.

**Review by boks1971 — COMMENTED** [review]
Looks good @cnderrauber . Yeah, need the `PeerID` bit like David mentioned.

File: pkg/sfu/buffer/buffer.go (hunk 1 above)
Comment (cnderrauber) [reply pkg/sfu/buffer/buffer.go]: sounds good. renamed

File: pkg/sfu/downtrack.go (hunk 2 above)
Comment (cnderrauber) [reply pkg/sfu/downtrack.go]: misunderstand it. revert change of peerid

**Review by boks1971 — APPROVED** [review]

**Review by davidzhao — APPROVED** [review]

**Pull request merged**